### PR TITLE
Added Appveyor integration

### PR DESCRIPTION
**Pull request by jdeschenes**
I added appveyor integration so that the unit test are built on windows as well as linux, macosx. This should help out your support for windows.

The script was tested on my own fork, but you will have to do a small setup on your own.

Before accepting this pull request, you will need to change the badges address in the README. If you have done that, you can let me know and I can change it in this PR.

Most of this is shamelessly taken from the pandas repo.

### CODE REVIEW

**Comment by jdeschenes** [comment]
It looks like the travis build is failing with a permission check. Is there anything I can do on my side to fix this?

**Comment by jwiggins** [comment]
I believe the Travis build is failing because the files don't have execute permissions.

`$ chmod +x ci/*.sh` should fix it.

File: README.rst [hunk 1]
```diff
@@ -9,6 +9,10 @@ celiagg
    :target: https://travis-ci.org/celiagg/celiagg
    :alt: Build status
 
+.. image:: https://ci.appveyor.com/api/projects/status/1d7wb65intsxrn90
+   :target: https://ci.appveyor.com/project/celiagg/celiagg
```
Comment (jwiggins) [inline README.rst]: Sorry, just noticed this should be: https://ci.appveyor.com/project/jwiggins/celiagg

Comment (jdeschenes) [inline README.rst]: Ok, I will wait until the travis build is ready before pushing this change in.

**Comment by jdeschenes** [comment]
Alright, the travis build passed. Let me know if you have any more things you want changed.

**Comment by jwiggins** [comment]
Nope. Looks like Travis and AppVeyor are happy now! Thanks!

**Pull request merged**